terraform/config_production.tfvars

```diff
@@ -1,6 +1,7 @@
 region                                = "eu-west-2"
 environment                           = "production"
-base_domain                           = "tradetesting.net"
+base_domain                           = "tradetesting.net" # trade-tariff.service.gov.uk
+frontend_base_domain                  = "tradetesting.net" # www.trade-tariff.service.gov.uk
```

Comment:
We'll have the root working after Saturday's migration, so do we need this?

Reply:
Its a noop atm